operators/multiclusterobservability/controllers/multiclusterobservability/multiclusterobservability_status.go

```diff
@@ -393,8 +443,27 @@ func newFailedCondition(reason string, msg string) *mcoshared.Condition {
 func newMetricsDisabledCondition() *mcoshared.Condition {
 	return &mcoshared.Condition{
 		Type:    "MetricsDisabled",
-		Status:  "True",
+		Status:  metav1.ConditionTrue,
 		Reason:  "MetricsDisabled",
 		Message: "Collect metrics from the managed clusters is disabled",
 	}
 }
+
+func newMCOADegradedCondition(missing []string) *mcoshared.Condition {
+	tmpl := "MultiCluster-Observability-Addon degraded because the following CRDs are not installed on the hub: %s"
+
+	var missingVersions []string
+	for _, name := range missing {
+		version := config.GetMCOASupportedCRDVersion(name)
+		missingVersions = append(missingVersions, fmt.Sprintf("%s(%s)", name, version))
+	}
+
+	msg := fmt.Sprintf(tmpl, strings.Join(missingVersions, ", "))
+
+	return &mcoshared.Condition{
+		Type:    reasonMCOADegraded,
```

Comment:
This was where the problem was. I tried refactoring the way the status was being generated but it made the PR bigger than what's necessary for this. In the future if we want we'll refactor it